packages/i18n/locales/en/translation.json

```diff
@@ -215,6 +218,8 @@
     "failedParsingCoreVersion": "Failed to determine core version.",
     "failedToFindGovernanceProposal": "Failed to find governance proposal with ID {{id}}.",
     "failedToGetTokenInfo": "Failed to get token info. Please enter a valid {{tokenType}} contract address.",
+    "failedToLoadMembersDescription": "Members failed to load. Check your internet connection or try again later. This can happen if the blockchain nodes are down.",
```

Comment:
I think we can get rid of the "blockchain nodes are down." It's not really actionable feedback for users.